composer.json

```diff
@@ -21,18 +21,16 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "conekta/conekta-php": "~2.0.4",
-        "illuminate/filesystem": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*",
-        "illuminate/support": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*",
-        "nesbot/carbon": "~1.0",
-        "symfony/http-kernel": "~2.6|~3.0"
+        "php": "^7.1",
```

Comment:
This change will automatically cause travis to fail when trying to run the inspection for php versions < 7.1, If you're changing this it will also be necessary to remove the first to elements of the PHP travis versions to inspect.